docs/FAQ.md

```diff
@@ -3,49 +3,102 @@ title: eScriptorium Tutorial - FAQ
+> - [Peter Stokes](https://www.ephe.psl.eu/peter-stokes) (Co-Director)
+> - Benjamin Kiessling (machine vision engineer, responsible for [Kraken](https://kraken.re/))
+> - Robin Tissot (lead project engineer for eScriptorium)
+> 
+> In addition to this are several other contributors, including a team from [Teklia](https://teklia.com/), and from the [Open Islamicate Texts Initiative](https://openiti.org/) at the University of Maryland and Northeastern University, among others.
+>
+> If you have questions or need help then you can try contacting us (follow the links for the names above to find contact details), but a more effective means might be to join [our community on Gitter](https://gitter.im/escripta/escriptorium) and ask your questions there.
+>
+> Finally, if you want to contact the eScriptorium team then please do not use the 'Contact' form on the website, as this goes to the local admins for the site and not to us (see [I contacted you via the eScriptorium website: why didn't you reply?](https://gitlab.com/scripta/escriptorium/-/wikis/I-contacted-you-via-the-eScriptorium-website:-why-didn't-you-reply%3F)).
 
 ## Is eScriptorium accessible offline?
 
+If you use an instance of eScriptorium hosted on a web server, no, you cannot access eScriptorium offline.
+
+If you use a [local deployment](https://gitlab.com/scripta/escriptorium/-/wikis/docker-install) of eScriptorium (installed on your machine and hosted by a virtual server), yes, you can access eScriptorium offline.
+
 ## Can I synchronize a local eScriptorium application with a server?
 
+eScriptorium is a decentralized software, which means that there are as many databases as there are instances of eScriptorium. It is impossible to synchronise two instances, unless they share the same database, which is unlikely.
+
+You can, however, update the content of your projects and documents from one instance to another with the help of the [export](walkthrough_export.md) and [import](walkthrough_import.md) features. For this, you need to have an account on both the source and the target instances.
+
 ## Who can I contact to request a new feature or signal a bug?
 
+If you want to provide feedback on the application, you can contact the eScriptorium developper team with their [Gitter community](https://gitter.im/escripta/escriptorium). You can also open [tickets](https://gitlab.com/scripta/escriptorium/-/issues) in the Gitlab repository hosting the source code of the application. If you want to offer a bug fix or contribute to the project, you can check the ["Contributing" page](https://gitlab.com/scripta/escriptorium/-/wikis/contributing).
+
+!!! Note
+    If you use eScriptorium on a server, we recommand identifying and contacting the owners of the application/server before reporting a bug to the eScriptorium developpement team. They may have deactivated some already existing features or be aware of a bug caused by the infrastructure rather than the application itself.
```

Comment:
"development" is written with only one 'p'
I think it should be "are aware of a bug ..." instead of "be aware"